docs/admin/system-information.rst

```diff
@@ -61,8 +61,8 @@ The result has at most 1 row::
 Cluster license
 ---------------
 
-The ``sys.cluster.license`` expression returns information about the currently
-registered license.
+The ``sys.cluster.license`` :ref:`expression <gloss-expression>` returns
+information about the currently registered license.
```

Comment:
Unrelated to your changes, but seems like we missed to add a deprecation note here with the license change to APL? Also in the table below we should note that the values are NULL for CrateDB >= 4.5.

Reply:
@seut added a new change to address this (see 20b9dad)